common/src/View/InputEvent.cpp

```diff
@@ -296,20 +296,20 @@ namespace TrenchBroom {
             }
         }
         
-        QPointF InputEventRecorder::scrollLinesForEvent(const QWheelEvent* qtEvent) {
-            // TODO: support pixel scrolling via qtEvent->pixelDelta()?
+        QPointF InputEventRecorder::scrollLinesForEvent(const QWheelEvent& qtEvent) {
+            // TODO: support pixel scrolling via qtEvent.pixelDelta()?
             const int linesPerStep = QApplication::wheelScrollLines();
-            const QPointF angleDelta = QPointF(qtEvent->angleDelta()); // in eighths-of-degrees
+            const QPointF angleDelta = QPointF(qtEvent.angleDelta()); // in eighths-of-degrees
             constexpr float EighthsOfDegreesPerStep = 120.0f; // see: https://doc.qt.io/qt-5/qwheelevent.html#angleDelta
 
             const QPointF lines = (angleDelta / EighthsOfDegreesPerStep) * linesPerStep;
             return lines;
         }
 
-        void InputEventRecorder::recordEvent(const QWheelEvent* qtEvent) {
+        void InputEventRecorder::recordEvent(const QWheelEvent& qtEvent) {
             // These are the mouse X and Y position, not the wheel delta
-            const int posX = qtEvent->x();
-            const int posY = qtEvent->y();
+            const auto posX = qtEvent.x();
+            const auto posY = qtEvent.y();
```

Comment:
rather not use auto here

Reply:
Now has a static_cast next to it, so the type is clear.